watertap/examples/chemistry/tests/test_pH_dependent_solubility.py

```diff
@@ -1223,8 +1225,13 @@ def run_case3(xOH=1e-7/55.2, xH=1e-7/55.2, xCaCO3=1e-20, xCaOH2 = 1e-20, xCa=1e-
     assert isinstance(model.fs.unit.control_volume.properties_in[0.0].scaling_factor, Suffix)
 
     ## ==================== END Scaling for this problem ===========================
-
-    model.fs.unit.initialize(optarg=solver.options, outlvl=idaeslog.DEBUG)
+    #   Loosen the tolerances for initialization stage by passing a temporary
+    #       config dict replacing default values (this does so without any
+    #       changes to the global solver options in WaterTAP)
+    temp_config = {"constr_viol_tol": 1e-5,
+                    "tol": 1e-5,
+                    "nlp_scaling_method": "user-scaling"}
```

Comment:
```suggestion
                   }
```
This option is already applied by the ipopt-watertap solver.